pkgs/development/python-modules/poetry/default.nix

```diff
@@ -36,7 +36,7 @@ in buildPythonPackage rec {
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0y528095njf28isbcp5iwbf12j67xhxnrkac93pws0zy133v7kc1";
+    sha256 = "02h387k0xssvv78yy82pcpknpq4w5ym2in1zl8cg9r5wljl5w6cf";
```

Comment:
why this change ?

Reply:
because it's wrong for me. Try doing `nix-build -A python3Packages.poetry.src --check` and you should get the same

Reply:
hydra also agrees https://hydra.nixos.org/eval/1577793?filter=poetry&compare=1577743&full=

Reply:
https://hydra.nixos.org/build/115163885/nixlog/1

Reply:
likely, poetry re-uploaded the poetry package with different contents. I don't think pypi is immutable (at least I've seen removals before)

Reply:
I checked indeed. Sad it's not immutable :( I hope it's a LGTM update on their side, it doesn't seem possible to check the different uploads from https://pypi.org/project/poetry/1.0.5/